src/pages/index.js

```diff
@@ -90,24 +131,49 @@ const previewModalCloseBtn = previewModal.querySelector(
 const cardTemplate = document.querySelector("#card-template");
 const cardList = document.querySelector(".cards__list");
 
+let selectedCard, selectedCardId;
+
+function handleLike(evt, id) {
+  evt.preventDefault();
+
+  const likeButton = evt.target;
+
+  const isLiked = likeButton.classList.contains("card__like-btn_liked");
+
+  api
+    .changeLikeStatus(id, !isLiked)
+    .then(() => {
+      likeButton.classList.toggle("card__like-btn_liked");
+    })
+    .catch(console.error);
+}
+
 //card information
 function getCardElement(data) {
   const cardElement = cardTemplate.content
     .querySelector(".card")
     .cloneNode(true);
-
+  0;
   const cardNameElement = cardElement.querySelector(".card__title");
   const cardImageElement = cardElement.querySelector(".card__image");
   const cardLikeButton = cardElement.querySelector(".card__like-btn");
   const cardDelete = cardElement.querySelector(".card__delete");
 
+  const isLiked =
+    Array.isArray(data.likes) &&
+    data.likes.some((like) => like._id === api.userId);
```

Comment:
* When I reload the page, all **red** hearts become **white** again. Please, check `isLiked`



```js
  const isLiked = data.isLiked

```